website/components/subnav/style.css

```diff
@@ -0,0 +1,15 @@
+/* TEMPORARY - only here until all sites convert to full width nav */
+
+.g-constrained-nav {
```

Comment:
As I look at this implementation further, would it be cleaner to style `.g-subnav` directly? Would it allow you to drop the `sticky`, `top`, and `z-index` props? There have been issues with `z-index` in the past, so I’d prefer it work the other way. What do you think? Either way, I’ll match your implementation.

```pcss
.g-subnav {
  & .g-subnav-inner {
    @media (width < 1288px) {
      padding: 0 32px;
    }

    @media (width >= 1288px) {
      padding: 0 calc(50% - 1288px / 2 + 32px);
    }
  }
}
```

Reply:
We've been having a bunch of conversations about whether to allow classNames to be passed in. @zchsh Tagging you here as another example. 

I was thinking that we could target the `.g-subnav` directly, but I added the container in there for clarity so we know it's being modified. We could take it out though and add a comment in it's place.

Reply:
I think adding a comment to describe the customization is a solid idea all on its own! My hope is to avoid contributing position or z-index.

Reply:
Would this work?
[style.css](https://github.com/hashicorp/vault/blob/a56d14268e273c8d72785ee6e843668b076488cc/website/components/subnav/style.css#L3)